droopler.install

```diff
@@ -1666,3 +1666,44 @@ function droopler_update_8123(&$sandbox) {
     \Drupal::messenger()->addMessage($sandbox['current'] . ' paragraphs processed, ' . $sandbox['changed'] . ' changed.');
   }
 }
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field in Reference Content Paragraph.
+ */
+function droopler_update_8124() {
+  /** @var Drupal\d_update\Updater $updater */
+  $updater = \Drupal::service('d_update');
+
+  $values['droopler__droopler_update_8124_1'] = update_modules_configuration(['d_p_reference_content'], '8124');
+  $updater->checklist()->saveProgress(['v2.2' => $values]);
+}
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field saved in database.
+ */
+function droopler_update_8125(&$sandbox) {
+  $limit = 25;
+  $entities = \Drupal::entityQuery('paragraph')->condition('type', 'd_p_reference_content')->execute();
+  if (!isset($sandbox['current'])) {
+    $sandbox['current'] = 0;
+    $sandbox['max'] = count($entities);
+  }
+  $batch = array_slice($entities, $sandbox['current'], $limit);
```

Comment:
"ciekawe" podejście, pobierasz tu wszystkie ID i dzielisz w arrayce.. o tyle dobrze, że to tylko ID i raczej nie powinno być ich dużo w bazie, to w końcu tylko `d_p_reference_content`

Reply:
Poprawione